examples/formatting-integer/profile.json

```diff
@@ -7,7 +7,7 @@
 		{ "constraints": [
 			{ "not": { "field": "an_integer", "is": "null" } },
 
-			{ "field": "an_integer", "is": "ofType", "value": "numeric" },
+			{ "field": "an_integer", "is": "ofType", "value": "decimal" },
```

Comment:
Should this be 'integer'?

Reply:
Since it had a granularity on it I did the straight swap to decimal. Both lines could be replaced with is an integer now?